README.md

```diff
@@ -13,9 +13,44 @@ Build and install the provider so that Terraform can use it:
 make install
 ```
 
+## Test
+
+### Unit tests
+
+Unit tests run the provider against a `localhost_apiserver` that serves data read from `/synthetics/test-data.json`
+
+This allows to:
+- avoid the necessity of providing valid API credentials
+- avoid creating resources on remote server
+- make the test results more reliable
+
+
+To run unit tests:
+```bash
+make test
+```
+
+This will:
```

Comment:
```suggestion
Running `make test` will:
```
Then `To run unit tests:` above can be removed. And we already have `make test` in development steps list

Reply:
Addressed: f1232530a413c31d588d1b567d9b1b5e007746be